modules/web-cli/assets/js/core/backwards-compatibility/commands.js

```diff
@@ -1,4 +1,3 @@
-import Helpers from 'elementor-api/utils/helpers';
 import Module from 'elementor-assets-js/modules/imports/module.js';
```

Comment:
Please exclude from the PR

Reply:
https://github.com/elementor/elementor/pull/18421/commits/c6c7d4cf0175ca0bcc9c44d526f4d23945646a98